documentation/tutorials/get-started-with-phoenix.md

```diff
@@ -0,0 +1,506 @@
+mix archive.install hex phx_new
+```
+
+And then run the project generator. **NOTE: DO NOT run `mix ecto.create`** we will do this the Ash way later.
+
+```bash
+mix phx.new my_ash_phoenix_app
+```
+
+This will then generate a project boilerplate and ask if you want to fetch and install dependencies.
+**Answer 'Yes' to 'Fetch and install dependencies? [Yn]'**.
+
+Your terminal should now look very similar to this:
+
+```bash
+* creating phoenix_helpdesk/priv/static/images/phoenix.png
+* creating phoenix_helpdesk/priv/static/favicon.ico
+
+Fetch and install dependencies? [Yn] y
+* running mix deps.get
+* running mix deps.compile
+
+We are almost there! The following steps are missing:
+
+    $ cd phoenix_helpdesk
+
+Then configure your database in config/dev.exs and run:
+
+    $ mix ecto.create
+
+Start your Phoenix app with:
+
+    $ mix phx.server
+
+You can also run your app inside IEx (Interactive Elixir) as:
+
+    $ iex -S mix phx.server
+```
+
+Please `cd` into the project directory, but again **DO NOT RUN `mix ecto.create`.**
+
+### Add Dependencies
+
+We now need to add Ash, AshPhoenix and AshPostgres to our Phoenix project.
+First lets add the dependencies to the `deps` function in our `mix.exs`:
+
+```elixir
+# mix.exs
+
+  def deps do
+    [
+      {:phoenix, "~> 1.6.15"},
+      # ...
+      {:ash, "~> 2.5.10"},
+      {:ash_postgres, "~> 1.3.6"},
+      {:ash_phoenix, "~> 1.1"},
+      # If using ElixirLS then including elixir_sense to enable Ash auto-complete
+      {:elixir_sense, github: "elixir-lsp/elixir_sense", only: [:dev, :test]}
+    ]
+  end
+```
+
+Each Ash package also has its own formatter extension, include these in your `.formatter.exs` to ensure consistent formatting.
+
+```elixir
+# .formatter.exs
+[
+  import_deps: [:ecto, :phoenix, :ash, :ash_phoenix, :ash_postgres],
+  inputs: ["*.{ex,exs}", "priv/*/seeds.exs", "{config,lib,test}/**/*.{ex,exs}"],
+  subdirectories: ["priv/*/migrations"]
+]
+```
+
+Now in the terminal install these new dependencies.
+
+```bash
+mix deps.get
+```
+
+### Use `AshPostgres.Repo` and Create the Database
+
+Swap `Ecto.Repo` for `AshPostgres.Repo`
+Change your repo module to look like this:
+
+```elixir
+# lib/my_ash_phoenix_app/repo.ex
+
+defmodule MyAshPhoenixApp.Repo do
+  use AshPostgres.Repo, opt_app: :my_ash_phoenix_app
+
+  # installs postgres extensions that ash commonly uses
+  def installed_extensions do
+    ["uuid-ossp", "citext"]
+  end
+end
+```
+
+`AshPostgres.Repo` enriches your repo with additional AshPostgres specific behaviour, but is essentially a thin wrapper around `Ecto.Repo`.
+
+After swapping `AshPostgres.Repo` in you are safe to create the database using:
+
+```
+> mix ash_postgres.create
+
+Compiling 17 files (.ex)
+Generated my_ash_phoenix_app app
+The database for MyAshPhoenixApp.Repo has been created
+```
+
+### Edit Config
+
+We need to specify the Ash API's that our application uses and some config for backwards compatibility that will be removed later.
```

Comment:
Don't think `API's` should have an apostrophe.

Reply:
addressed: [5c2cb7f](https://github.com/ash-project/ash/pull/504/commits/5c2cb7fdf6732e6b12681b9cd23819322f0baa87)